.env-example

```diff
@@ -4,3 +4,9 @@ DATABASE_NAME=your database name
 DATABASE_USER=your user name
 DATABASE_PASSWORD=your user password
 DATABASE_HOST=your host
+
+
+# Cloudnary environment configurations
+export CLOUDINARY_NAME=''
```

Comment:
Remember to add these environment variables on travis-ci.org and travis-ci.com to ensure your build passes.